source/frontend/src/components/common/form/FastDatePicker.tsx

```diff
@@ -157,9 +157,6 @@ const StyledGroup = styled(Form.Group)`
   .react-datepicker__view-calendar-icon input {
     padding: 0.8rem 1.2rem 0.8rem 1.2rem;
   }
-  .react-datepicker-wrapper {
```

Comment:
This allows datepickers to grow to fit available space - which allows horizontal alignment on the filter screens:

![image](https://github.com/user-attachments/assets/e4674965-aecb-4c47-84e1-816450bf2e1b)